// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

import "components/feed/core/proto/v2/wire/web_feed_id.proto";

option optimize_for = LITE_RUNTIME;

message WebFeedIdentifierToken {
  oneof id_oneof {
    WebFeedId web_feed_id = 4;
  }
}
